services/tokei/tokei-github-lines.service.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { metric } = require('../text-formatters')
+const { nonNegativeInteger } = require('../validators')
+const { BaseJsonService } = require('..')
+
+const schema = Joi.object({
+  lines: nonNegativeInteger,
+}).required()
+
+const documentation = `
+<p>
+  The <code>provider</code> is the domain name of git host.
+  If no TLD is provided, <code>.com</code> will be added.
+  For example, setting <code>gitlab</code> or <code>bitbucket.org</code> as the provider also works.
+</p>
+`
+
+module.exports = class LinesOfCode extends BaseJsonService {
+  static category = 'size'
+
+  static route = { base: 'tokei/l', pattern: ':provider/:user/:repo' }
```

Comment:
I think it'd be better to use a more complete noun here, especially since elsewhere in Shields `l` has other meanings (licenses) 
```suggestion
  static route = { base: 'tokei/lines', pattern: ':provider/:user/:repo' }
```